### PR TITLE
fix(ui5-label): fix width for italic labels

### LINKED ISSUE #1619: italic style text in ui5-label get truncated
If ui5-label components has font-style: italic, its text will be truncated.
Screen Shot 2020-05-15 at 2 21 35 PM

in above screenshot, the label text should be "**dddd**", but it looks like "**ddda**"

it is easy to reproduced on ui5 web component playground as well, just add "font-style: italic" to any ui5-label, you will see the problem:
https://sap.github.io/ui5-webcomponents/playground/components/Label/

Screen Shot 2020-05-15 at 2 29 13 PM

### PR DESCRIPTION
**Pull request by fifoosid**
Fixes #1619

### CODE REVIEW

File: packages/main/src/themes/Label.css [hunk 1]
```diff
@@ -50,6 +55,11 @@
 	font-size: 1.25rem;
 	font-weight: bold;
 	position: relative;
+	font-style: normal;
 	vertical-align: middle;
 	line-height: 0;
+}
+
+:host([required][show-colon]) .ui5-label-required-colon:after {
```
Comment (ilhan007) [inline packages/main/src/themes/Label.css:63]: I cant see the RTL equivalent, do we need margin-right: 2px in RTL?

File: packages/main/src/themes/Label.css [hunk 2]
```diff
@@ -20,6 +20,11 @@
 	overflow: hidden;
 }
 
+.ui5-label-text-wrapper:after {
+	padding-right: 2.5px;
```
Comment (ilhan007) [inline packages/main/src/themes/Label.css:24]: do we need to mirror the padding in RTL

File: packages/main/src/themes/Label.css (hunk 1 above)
Comment (fifoosid) [reply packages/main/src/themes/Label.css:63]: Done

File: packages/main/src/themes/Label.css (hunk 2 above)
Comment (fifoosid) [reply packages/main/src/themes/Label.css:24]: Not needed

File: packages/main/src/themes/Label.css [hunk 3]
```diff
@@ -20,6 +20,11 @@
 	overflow: hidden;
 }
 
+bdi{
+    content: "";
+	padding-right: 2.5px;
```
Comment (ilhan007) [inline packages/main/src/themes/Label.css:25]: Should the bdi also gets its RTL mirrored padding?
And in general all sizes is good to be in rems

Comment (fifoosid) [reply packages/main/src/themes/Label.css]: No it is not needed.
Size is in rems now

File: packages/main/src/themes/Label.css [hunk 4]
```diff
@@ -50,6 +55,18 @@
 	font-size: 1.25rem;
 	font-weight: bold;
 	position: relative;
+	font-style: normal;
 	vertical-align: middle;
 	line-height: 0;
+}
+
+:host([required][show-colon]) .ui5-label-required-colon:after {
+	margin-right: 0;
```
Comment (ilhan007) [inline packages/main/src/themes/Label.css:64]: The margin here and below are still 2px, let's change them to 0.125rem

**Pull request merged**